public/js/revue_app.js

```diff
@@ -0,0 +1,68 @@
+(function(){
+  var app = angular.module('revue', []);
+  app.controller('LogsController', function($scope, $http){
+    $scope.logs = [];
+
+    $http.get('/logs')
+
+    .success(function(data) {
+      $scope.logs = data;
+    })
+
+    .error(function(data) {
+      console.log("Error when getting the json at /logs");
+    });
+  });
+  app.controller('MostActiveController', function($scope, $http){
+    $scope.most_active = [];
+
+    $http.get('/most_active')
+
+    .success(function(data) {
+      $scope.most_active = data;
+    })
+
+    .error(function(data) {
+      console.log("Error when getting the json at /most_active");
+    });
+
+  });
+
+
+
+  app.controller('MostActiveController', function($scope, $http){
+
+    var chart_width = 940;
```

Comment:
Identifier 'chart_width' is not in camel case.<br>'chart_width' is defined but never used.